src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -98,7 +101,7 @@ private void setup() {
         }).collect(Collectors.toList());
     }
 
-    private List<BrAPIGermplasm> fetchProgramGermplasm(UUID programId) throws ApiException {
+    private Map<String, BrAPIGermplasm> fetchProgramGermplasm(UUID programId) throws ApiException {
```

Comment:
Method header comment detailing the output format, key = breeding insight germplasm id, value = formatted BrAPI Germplasm would be useful. It's not clear reading the method what the key is.